store/tikv/txn.go

```diff
@@ -347,8 +348,10 @@ func (txn *tikvTxn) LockKeys(ctx context.Context, lockCtx *kv.LockCtx, keysInput
 				}
 			}
 		}
+		if lockCtx.LockKeysCount != nil {
+			*lockCtx.LockKeysCount += int32(len(keys))
```

Comment:
Why not add the metrics directly?

Reply:
This may be used in slow log or other statistic system tables